Sources/App/githubRoutes.swift

```diff
@@ -53,4 +53,22 @@ public func githubRoutes(router: GithubCommandRouter) throws {
             on: req
         ).transform(to: .ok)
     }
+    
+    router.command("hello") { (req, webhook) -> Future<HTTPStatus> in
```

Comment:
@twof You forgot the bot prefix here, I think any message starting with `hello` will trigger now

Reply:
https://github.com/vapor/routing/pull/66#issuecomment-469965409

Nah the routes are exact matches